components/Event.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react';
+import demoEvents from '../demoEvents.json';
+import Image from 'next/image';
+interface Props {
+  event: typeof demoEvents[0];
+}
+
+const Event: FC<Props> = ({ event }) => {
```

Comment:
Since the _event_ object is being used quite frequently shouldn't we be destructuring its different properties here?

Reply:
I think once we have a fully developed API, what we can do is instead of passing the whole object to the component, we can just loop through event ids and call the corresponding API to fetch that object from within the component. I just kinda made it like that for now so we can see the events. I can mock what that would look like if we want to look into doing something like that now